app/views/start/index.html.slim

```diff
@@ -0,0 +1,31 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h1 class="govuk-heading-xl" = t(".efe_heading")
+
+    p class="govuk-body" = t(".legal_aid_provider")
+
+    p class="govuk-body"= t(".non_provider_check_link_html")
+
+    h2 class="govuk-heading-m"= t(".before_start_heading")
+
+    p class="govuk-body"= t(".required_information")
+
+    .govuk-list
+      ul class="govuk-list govuk-list--bullet"
+        li = t(".item_1")
+        li = t(".item_2")
+        li = t(".item_3")
+
+      = link_to start_button_label(:start_button),
+        "/status",
```

Comment:
One other thing: it would be really cool if this linked to `new_estimate_path`, as that is now able to call CFE, generate an assessment ID and move the user into the flow.